src/h5web/vis-packs/core/heatmap/ColorBar.module.css

```diff
@@ -38,6 +41,14 @@
   overflow: visible;
 }
 
+.colorBar:not([data-horizontal]) > .axis {
+  border-top: 1px solid transparent; /* fix alignment of ticks */
+}
+
+.colorBar[data-horizontal] > .axis {
+  border-left: 1px solid transparent; /* fix alignment of ticks */
+}
```

Comment:
The outline fix was smart, but it made the color bar visually bigger than its container, which isn't ideal. What do you think of this? The problem is that the first tick is vertically (or horizontally) positioned in the middle of its theoretical position. Sometimes, because of rounding, it ends up being rendered one pixel too high. By adding a 1px transparent border, I move every tick down by 1px, which gives good-enough alignment in all cases.

![image](https://user-images.githubusercontent.com/2936402/128699488-cd460f5e-0b74-4c5d-9f80-216ac6904393.png)

![image](https://user-images.githubusercontent.com/2936402/128699523-283edba2-e022-4f0e-84d7-612eb2a846f1.png)

![image](https://user-images.githubusercontent.com/2936402/128699537-9c0b0db9-6457-441a-996e-f0c9daaab080.png)

Reply:
Clever :100:

Reply:
Could we use `padding` or `margin` instead of `border` ? It would make more sense, no ?

Reply:
Hmm, I thought visx would ignore the padding when positioning the ticks, and that the margin would push the last tick outside of the colorbar, but I was wrong: both seem work fine. I went with the padding.